src/Views/Home.js

```diff
@@ -14,9 +13,6 @@ class Home extends Component {
   render() {
     return (
       <div>
-        <header>
-          <Header />
-        </header>
         {/* Top section */}
         <div style={{ border: 'solid black', padding: '2vw' }}>
           <Grid stackable columns={1}>
```

Comment:
Same case, with these two lines.

Reply:
This is also due to linting, but it's a different case. The `<Header />` tag is self-closing and has no children, so it must have this spacing. 

https://github.com/airbnb/javascript/tree/master/react#spacing

Reply:
The first header tag is there just a semantic element to identify it later on.

Reply:
This is helpful, good to know for the future. But I was talking about `<div style={{ border: 'solid black', padding: '2vw' }}>` and `<Grid stackable columns={1}>` the braces-spacing.

Reply:
The style attribute in React is just a JavaScript object, that's why it uses the double curly braces, as such, it needs the spacing;  for the second piece of code, columns is a prop for the Grid component, that makes it jsx in which case the spacing is not required.